integration-tests/features/component_analysis_v2.feature

```diff
@@ -0,0 +1,246 @@
+      Then I should find recommended version <recommended-version> in the component analysis
+      Then I should find one or more vulnerabilities in result with valid attributes
+      And I should find CVE report <cve> with score <score> in the component v2 analysis
+      
+     
+      
+
+
+     Examples: EPV
+     | ecosystem  | package             | version   | recommended-version | cve                         | score  |
+     | npm        | ejs                 | 1.0.0     | 3.1.3               | SNYK-JS-EJS-10218           | 8.1    |
+     | npm        | marked              | 0.3.5     | 1.1.0               | SNYK-JS-MARKED-10377        | 7.5    |
+     | npm        | st                  | 0.2.4     | 2.0.0               | SNYK-JS-ST-10820            | 4.3    |
+     | npm        | npmconf             | 0.0.24    | 2.1.3               | SNYK-JS-NPMCONF-12143       | 7.4    |
+     | npm        | moment              | 2.15.1    | 2.25.3              | SNYK-JS-MOMENT-10841        | 3.7    |
+     | npm        | mongoose            | 4.2.4     | 5.9.15              | SNYK-JS-MONGOOSE-10081      | 5.1    |
+     | maven      | io.vertx:vertx-core | 3.5.3     | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-72443     | 6.5    |
+     | maven      | io.vertx:vertx-core | 3.5.3.CR1 | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-72443     | 6.5    |
+     | maven      | io.vertx:vertx-core | 3.5.2     | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-72443     | 6.5    |
+     | maven      | io.vertx:vertx-core | 3.5.2.CR3 | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-32470     | 5.3    | 
+     | maven      | io.vertx:vertx-core | 3.5.2.CR2 | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-32470     | 5.3    | 
+     | maven      | io.vertx:vertx-core | 3.5.2.CR1 | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-32470     | 5.3    |
+     | pypi       | flask               | 0.12      | 1.1.2               | SNYK-PYTHON-FLASK-42185     | 7.5    |
+     | pypi       | flask               | 0.12.1    | 1.1.2               | SNYK-PYTHON-FLASK-42185     | 7.5    |
+     | pypi       | flask               | 0.12.2    | 1.1.2               | SNYK-PYTHON-FLASK-42185     | 7.5    |
+     | pypi       | numpy               | 1.16.0    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.4    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.3    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.2    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.1    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | requests            | 2.19.1    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
+     | pypi       | requests            | 2.19.0    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
```

Comment:
any specific reasons for testing it with several versions of the same package with the same vulnerability? The results would be same for all these cases.

Reply:
i have followed the way how pavel has written for V1 
same package different version should give you same recommended version
if not fail

Reply:
as per my understanding, it doesnt make sense to me @prashbnair